### DISCUSSION TITLE
Welcome to our New Community!

### DISCUSSION THREAD

**Discussion by atrolex** [atrocore/atrocore#175] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions regarding AtroCore, AtroPIM, AtroDAM and other modules you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by atrolex** [comment]
Sorry to all, we have decided to remove discussions on the AtroPIM and other repository pages. We believe, it is better to have everything on one page. This is the page of AtroCore main repository.

**Reply by marcus-at-localhost** [reply]
Makes totally sense!
Is there a way to move previous discussions to this repo? I've seen it on other multi repo projects.
Cheers, Marcus.

**Reply by atrolex** [reply]
Indeed:) There is a posibility, thx Marcus! I will transfer everything.
